### DISCUSSION TITLE
Integration to home assistant

### DISCUSSION THREAD

**Discussion by moonraker69** [alex-konshin/f007th-rpi#19] [General]
HI, I saw this project and immediately needed to build and setup the HW with my raspberry to get hold of my temperature sensors that are around in my house. Great project also using the cheap RXB6 receiver is a good idea, other projects for the raspberry are using rtl-sdr which is much more expensive and also too complicated and power consuming for just temperature readings from my pov. Anyway now having this running I want to integrate this also to my other raspberry pi that runs Home assistant permanentely and I would like to remove the current "normal" Raspi in the game. But here comes the problem, I was not able to build this in Homeassistant OS which actually is not a full linux distribution, but more like a system that is able to run predefined docker containers, as I understand. So far it seems no RXB6 supported driver is yet available as a home assistant add-on or in HACS. I did play around and did a bit investigations but so far didn't understand yet if this is technically possible or how this could be done. Any suggestions, or would you even be interested in porting this to HA ? BR moonraker

**Comment by alex-konshin** [comment]
Sorry, my project was on hold, but I am going to resurrect it.
I need some back end for my multi zone HVAC system. I am looking at existing solutions to simplify implementation.
I am considering integration with Home Assistant but it is completely new for me. Yes, looks like it is a custom Linux OS with containers.
I need to investigate how I can implement new components and even use their own drivers like my gpio-ts. Looks like it will not be easy.
Otherwise, I will try just use the existing functionality of my utility to push sensor data to MQTT. I can see in the HA documentation that it has integration with MQTT. It this case it will not require any change in the utility.
